askama_shared/src/generator.rs

```diff
@@ -1697,16 +1697,20 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
         self.prepare_ws(ws);
     }
 
+    fn should_trim_ws(&self, ws: Option<Sign>) -> bool {
+        ws == Some(Sign::Minus) || self.automatic_block_trim && ws != Some(Sign::Plus)
```

Comment:
I feel like this could be clearer as a `match`?

```rust
match ws {
    Some(Sign::Minus) => true,
    Some(Sign::Plus) => false,
    None => self.automatic_block_trim,
}
```

Reply:
That is indeed much better.